test/Fields/field_multi_broadcast_fusion.jl

```diff
@@ -74,13 +74,9 @@ end
 
 function benchmark_kernel!(f!, X, Y, device)
     println("\n--------------------------- $(nameof(typeof(f!))) ")
-    trial = benchmark_kernel!(f!, X, Y, device)
+    trial = BenchmarkTools.@benchmark ClimaComms.@cuda_sync $device $f!($X, $Y)
```

Comment:
Shouldn't this be `ClimaComms.cuda_sync` (ie, not a macro)?

Reply:
I kept the macros around for backward compatibility: https://github.com/CliMA/ClimaComms.jl/blob/e62f126620791c09888009e04aa71ad7899ba149/src/devices.jl#L331-L334, but you're right that we should switch them over and eventually deprecate the macros.

Reply:
Ah, okay. Got it. Up to you if you want to do this now or in the future.